### ISSUE TITLE
[bug] Conan 1.31.1 does not work since urllib3 1.26.0 release

### ISSUE THREAD

**Issue by f18m** [conan-io/conan#8041]
### Environment Details (include every applicable attribute)
  * Operating System+version: Centos 7.8
  * Compiler+version:
  * Conan version: 1.31.1
  * Python version: 3.6.8

### Steps to reproduce (Include if Applicable)

pip3 install conan

```
Collecting conan
  Using cached https://files.pythonhosted.org/packages/77/2b/8359311c80c1fb61a8e6b93303d278d8282057f8ae632b8d5931449d83e1/conan-1.31.1.tar.gz
Requirement already satisfied: PyJWT<2.0.0,>=1.4.0 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: requests<3.0.0,>=2.8.1 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: urllib3!=1.25.4,!=1.25.5 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: colorama<0.5.0,>=0.3.3 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: PyYAML<6.0,>=3.11 in /usr/local/lib64/python3.6/site-packages (from conan)
Requirement already satisfied: patch-ng<1.18,>=1.17.4 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: fasteners>=0.14.1 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: six<=1.15.0,>=1.10.0 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: node-semver==0.6.1 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: distro<=1.5.0,>=1.0.2 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: future<0.19.0,>=0.16.0 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: pygments<3.0,>=2.0 in /usr/local/lib64/python3.6/site-packages (from conan)
Requirement already satisfied: deprecation<2.1,>=2.0 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: tqdm<5,>=4.28.1 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: Jinja2<3,>=2.9 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: python-dateutil<3,>=2.7.0 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: bottle<0.13,>=0.12.8 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: pluginbase>=0.5 in /usr/local/lib/python3.6/site-packages (from conan)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0,>=2.8.1->conan)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0,>=2.8.1->conan)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0,>=2.8.1->conan)
Requirement already satisfied: monotonic>=0.1 in /usr/local/lib/python3.6/site-packages (from fasteners>=0.14.1->conan)
Requirement already satisfied: packaging in /usr/local/lib/python3.6/site-packages (from deprecation<2.1,>=2.0->conan)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib64/python3.6/site-packages (from Jinja2<3,>=2.9->conan)
Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.6/site-packages (from packaging->deprecation<2.1,>=2.0->conan)
Installing collected packages: conan
  Running setup.py install for conan ... done
Successfully installed conan-1.31.1

```

conan --version

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 570, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 888, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 779, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (urllib3 1.26.0 (/usr/local/lib/python3.6/site-packages), Requirement.parse('urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1'), {'requests'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/conan", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3095, in <module>
    @_call_aside
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3079, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3108, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 572, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 585, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 779, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (urllib3 1.26.0 (/usr/local/lib/python3.6/site-packages), Requirement.parse('urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1'), {'requests'})

```

Just for check:
pip3 list | grep -E 'conan|urllib|requests'

I get

```
conan (1.31.1)
requests (2.24.0)
urllib3 (1.26.0)
```

**Comment by memsharded** [comment]
Hi @f18m, thanks for reporting this

I am not sure why we even explicitly constrain urllib3, I am doing https://github.com/conan-io/conan/pull/8042 to check

It seems that constrain comes from buggy urllib3 version that was filtered out in https://github.com/conan-io/conan/pull/5808. If requests is doing the right thing now, maybe we can skip it?

**Comment by dariuszparys** [comment]
The 1.31.1 version throws this in our packer build

```powershell
    azure-arm: Mode                LastWriteTime         Length Name
==> azure-arm: pkg_resources.ContextualVersionConflict: (urllib3 1.26.0 (c:\python37\lib\site-packages), Requirement.parse('urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1'), {'requests'})
    azure-arm: ----                -------------         ------ ----
    azure-arm: d-----       11/11/2020   7:31 AM                Conan
==> azure-arm:
    azure-arm:
    azure-arm: SUCCESS: Specified value was saved.
==> azure-arm: During handling of the above exception, another exception occurred:
==> azure-arm:
==> azure-arm: Traceback (most recent call last):
==> azure-arm:   File "C:\Python37\Scripts\conan-script.py", line 6, in <module>
==> azure-arm:     from pkg_resources import load_entry_point
==> azure-arm:   File "c:\python37\lib\site-packages\pkg_resources\__init__.py", line 3250, in <module>
==> azure-arm:     @_call_aside
==> azure-arm:   File "c:\python37\lib\site-packages\pkg_resources\__init__.py", line 3234, in _call_aside
==> azure-arm:     f(*args, **kwargs)
==> azure-arm:   File "c:\python37\lib\site-packages\pkg_resources\__init__.py", line 3263, in _initialize_master_working_set
==> azure-arm:     working_set = WorkingSet._build_master()
==> azure-arm:   File "c:\python37\lib\site-packages\pkg_resources\__init__.py", line 585, in _build_master
==> azure-arm:     return cls._build_from_requirements(__requires__)
==> azure-arm:   File "c:\python37\lib\site-packages\pkg_resources\__init__.py", line 598, in _build_from_requirements
==> azure-arm:     dists = ws.resolve(reqs, Environment())
==> azure-arm:   File "c:\python37\lib\site-packages\pkg_resources\__init__.py", line 791, in resolve
==> azure-arm:     raise VersionConflict(dist, req).with_context(dependent_req)
==> azure-arm: pkg_resources.ContextualVersionConflict: (urllib3 1.26.0 (c:\python37\lib\site-packages), Requirement.parse('urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1'), {'requests'})
==> azure-arm: Traceback (most recent call last):
==> azure-arm:   File "c:\python37\lib\site-packages\pkg_resources\__init__.py", line 583, in _build_master
==> azure-arm:     ws.require(__requires__)
==> azure-arm:   File "c:\python37\lib\site-packages\pkg_resources\__init__.py", line 900, in require
==> azure-arm:     needed = self.resolve(parse_requirements(requirements))
==> azure-arm:     raise VersionConflict(dist, req).with_context(dependent_req)
==> azure-arm: pkg_resources.ContextualVersionConflict: (urllib3 1.26.0 (c:\python37\lib\site-packages), Requirement.parse('urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1'), {'requests'})
```
Python 3.7.7 on Windows

**Comment by aheumaier** [comment]
This seems his seems to affect other conan version s as `1.30.X` and `1.29.x`   too .
Reproduce with : 

```shell
~$ python -m venv myenv
~$ pip install conan == 1.30.1
~$ source myenv/bin/activate
# (...)
  ERROR: requests 2.24.0 has requirement urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1, but you'll have urllib3 1.26.0 which is incompatible.
  Installing collected packages: urllib3, distro, six, monotonic, fasteners, patch-ng, tqdm, MarkupSafe, Jinja2, PyJWT, pygments, 
  PyYAML, colorama, bottle, pluginbase, chardet, idna, certifi, requests, future, pyparsing, packaging, deprecation, node-semver, 
  python-dateutil, conan
  Successfully installed Jinja2-2.11.2 MarkupSafe-1.1.1 PyJWT-1.7.1 PyYAML-5.3.1 bottle-0.12.18 certifi-2020.11.8 chardet-3.0.4 
  colorama-0.4.4 conan-1.30.1 deprecation-2.0.7 distro-1.5.0 fasteners-0.15 future-0.18.2 idna-2.10 monotonic-1.5 node-semver- 
  0.6.1 packaging-20.4 patch-ng-1.17.4 pluginbase-1.0.0 pygments-2.7.2 pyparsing-2.4.7 python-dateutil-2.8.1 requests-2.24.0 s 
  six-1.15.0 tqdm-4.51.0 urllib3-1.26.0

~$ pip list | grep -E 'conan|urllib|requests'
conan           1.30.1
requests        2.24.0
urllib3         1.26.0
```

**Comment by trend-laurence-cheng** [comment]
The workaround for me is downgrading urllib3 to `1.25.11`.

**Comment by dariuszparys** [comment]
After reading through the workaround this also works with the latest conan package

`pip install conan --upgrade --use-feature=2020-resolver`

**Comment by n-bes** [comment]
Log from windows gitlab-ci agent
```
[0KRunning with gitlab-runner 13.4.1 (e95f89a0)
[0;m[0K  on windows-shared-runners-manager 6QgxEPvR
[0;msection_start:1605099144:resolve_secrets
[0K[0K[36;1mResolving secrets[0;m
[0;msection_end:1605099144:resolve_secrets
[0Ksection_start:1605099144:prepare_executor
[0K[0K[36;1mPreparing the "custom" executor[0;m
[0;m[0KUsing Custom executor with driver autoscaler dev (671d969)...
[0;mCreating virtual machine for the job...
Virtual machine created!
section_end:1605099283:prepare_executor
[0Ksection_start:1605099283:prepare_script
[0K[0K[36;1mPreparing environment[0;m
[0;mRunning on PACKER-5F1153D4 via 
runner-6qgxepvr-wsrm-8d7cedee88a5b6d1a6e8...
section_end:1605099296:prepare_script
[0Ksection_start:1605099296:get_sources
[0K[0K[36;1mGetting source from Git repository[0;m
[0;m[32;1mFetching changes with git depth set to 50...[0;m
Initialized empty Git repository in C:/GitLab-Runner/builds/n.bes/cci-ci/.git/
[32;1mCreated fresh repository.[0;m
[32;1mChecking out 980b07b9 as gitlab-ci...[0;m
git-lfs/2.8.0 (GitHub; windows amd64; go 1.12.2; git 30af66bb)

[32;1mSkipping Git submodules setup[0;m
section_end:1605099323:get_sources
[0Ksection_start:1605099323:restore_cache
[0K[0K[36;1mRestoring cache[0;m
[0;mVersion:      13.1.1
Git revision: 6fbc7474
Git branch:   13-1-stable
GO version:   go1.13.8
Built:        2020-07-01T06:50:35+0000
OS/Arch:      windows/amd64
[32;1mChecking cache for gitlab-ci...[0;m
[0;33mFailed to extract cache[0;m
Runtime platform                                  [0;m  arch[0;m=amd64 os[0;m=windows pid[0;m=4032 revision[0;m=6fbc7474 version[0;m=13.1.1
[31;1mFATAL: file does not exist                        [0;m 
section_end:1605099335:restore_cache
[0Ksection_start:1605099335:step_script
[0K[0K[36;1mExecuting "step_script" stage of the job script[0;m
[0;m[0;33mWARNING: Starting with version 14.0 the 'build_script' stage will be replaced with 'step_script': https://gitlab.com/gitlab-org/gitlab-runner/-/issues/26426
[0;m[32;1m$ choco install -y cmake[0;m
Chocolatey v0.10.15
Installing the following packages:
cmake
By installing you accept licenses for the packages.

python3 v3.8.0 [Approved]
python3 package files install completed. Performing other installation steps.
Installing 64-bit python3...
python3 has been installed.
Installed to: 'C:\Python38'
  python3 can be automatically uninstalled.
Environment Vars (like PATH) have changed. Close/reopen your shell to
 see the changes (or in powershell/cmd.exe just type `refreshenv`).
 The install of python3 was successful.
  Software installed as 'exe', install location is likely default.

python v3.8.0 [Approved]
python package files install completed. Performing other installation steps.
 The install of python was successful.
  Software install location not explicitly set, could be in package or
  default install location if installer.

Chocolatey installed 2/2 packages. 
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Enjoy using Chocolatey? Explore more amazing features to take your
experience to the next level at
 https://chocolatey.org/compare
[32;1m$ $env:PATH+=";C:\Python38;C:\Python38\Scripts;C:\Program Files\CMake\bin"[0;m
[32;1m$ pip install --upgrade urllib3[0;m
Collecting urllib3
  Downloading https://files.pythonhosted.org/packages/7e/a7/746338eb8addda2e7662ee5e10a9f85150aba013cd610c9569c17146b914/urllib3-1.26.0-py2.py3-none-any.whl (136kB)
Installing collected packages: urllib3
Successfully installed urllib3-1.26.0
[32;1m$ pip install conan[0;m
WARNING: You are using pip version 19.2.3, however version 20.2.4 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.
Collecting conan
  Downloading https://files.pythonhosted.org/packages/77/2b/8359311c80c1fb61a8e6b93303d278d8282057f8ae632b8d5931449d83e1/conan-1.31.1.tar.gz (631kB)
Collecting PyJWT<2.0.0,>=1.4.0 (from conan)
  Downloading https://files.pythonhosted.org/packages/87/8b/6a9f14b5f781697e51259d81657e6048fd31a113229cf346880bb7545565/PyJWT-1.7.1-py2.py3-none-any.whl
Collecting requests<3.0.0,>=2.8.1 (from conan)
  Downloading https://files.pythonhosted.org/packages/45/1e/0c169c6a5381e241ba7404532c16a21d86ab872c9bed8bdcd4c423954103/requests-2.24.0-py2.py3-none-any.whl (61kB)
Requirement already satisfied: urllib3!=1.25.4,!=1.25.5 in c:\python38\lib\site-packages (from conan) (1.26.0)
Collecting colorama<0.5.0,>=0.3.3 (from conan)
  Downloading https://files.pythonhosted.org/packages/44/98/5b86278fbbf250d239ae0ecb724f8572af1c91f4a11edf4d36a206189440/colorama-0.4.4-py2.py3-none-any.whl
Collecting PyYAML<6.0,>=3.11 (from conan)
  Downloading https://files.pythonhosted.org/packages/7b/1d/3cf27312e162ff8bf3a9ae8c7472bb28ca6ae7f86e7130c23b798ec4c12a/PyYAML-5.3.1-cp38-cp38-win_amd64.whl (219kB)
Collecting patch-ng<1.18,>=1.17.4 (from conan)
  Downloading https://files.pythonhosted.org/packages/c1/b2/ad3cd464101435fdf642d20e0e5e782b4edaef1affdf2adfc5c75660225b/patch-ng-1.17.4.tar.gz
Collecting fasteners>=0.14.1 (from conan)
  Downloading https://files.pythonhosted.org/packages/18/bd/55eb2d6397b9c0e263af9d091ebdb756b15756029b3cededf6461481bc63/fasteners-0.15-py2.py3-none-any.whl
Collecting six<=1.15.0,>=1.10.0 (from conan)
  Downloading https://files.pythonhosted.org/packages/ee/ff/48bde5c0f013094d729fe4b0316ba2a24774b3ff1c52d924a8a4cb04078a/six-1.15.0-py2.py3-none-any.whl
Collecting node-semver==0.6.1 (from conan)
  Downloading https://files.pythonhosted.org/packages/08/51/6cf3a2b18ca35cbe4ad3c7538a7c3dc0cb24e71629fb16e729c137d06432/node_semver-0.6.1-py3-none-any.whl
Collecting distro<=1.5.0,>=1.0.2 (from conan)
  Downloading https://files.pythonhosted.org/packages/25/b7/b3c4270a11414cb22c6352ebc7a83aaa3712043be29daa05018fd5a5c956/distro-1.5.0-py2.py3-none-any.whl
Collecting future<0.19.0,>=0.16.0 (from conan)
  Downloading https://files.pythonhosted.org/packages/45/0b/38b06fd9b92dc2b68d58b75f900e97884c45bedd2ff83203d933cf5851c9/future-0.18.2.tar.gz (829kB)
Collecting pygments<3.0,>=2.0 (from conan)
  Downloading https://files.pythonhosted.org/packages/d8/cf/d7e48846e10ac4f8b89d489cec3dc9c2804a825f812c851559cfbcc251cc/Pygments-2.7.2-py3-none-any.whl (948kB)
Collecting deprecation<2.1,>=2.0 (from conan)
  Downloading https://files.pythonhosted.org/packages/b9/2a/d5084a8781398cea745c01237b95d9762c382697c63760a95cc6a814ad3a/deprecation-2.0.7-py2.py3-none-any.whl
Collecting tqdm<5,>=4.28.1 (from conan)
  Downloading https://files.pythonhosted.org/packages/93/3a/96b3dc293aa72443cf9627444c3c221a7ba34bb622e4d8bf1b5d4f2d9d08/tqdm-4.51.0-py2.py3-none-any.whl (70kB)
Collecting Jinja2<3,>=2.9 (from conan)
  Downloading https://files.pythonhosted.org/packages/30/9e/f663a2aa66a09d838042ae1a2c5659828bb9b41ea3a6efa20a20fd92b121/Jinja2-2.11.2-py2.py3-none-any.whl (125kB)
Collecting python-dateutil<3,>=2.7.0 (from conan)
  Downloading https://files.pythonhosted.org/packages/d4/70/d60450c3dd48ef87586924207ae8907090de0b306af2bce5d134d78615cb/python_dateutil-2.8.1-py2.py3-none-any.whl (227kB)
Collecting bottle<0.13,>=0.12.8 (from conan)
  Downloading https://files.pythonhosted.org/packages/e9/39/2bf3a1fd963e749cdbe5036a184eda8c37d8af25d1297d94b8b7aeec17c4/bottle-0.12.18-py3-none-any.whl (89kB)
Collecting pluginbase>=0.5 (from conan)
  Downloading https://files.pythonhosted.org/packages/3d/3c/fe974b4f835f83cc46966e04051f8708b7535bac28fbc0dcca1ee0c237b8/pluginbase-1.0.0.tar.gz (41kB)
Collecting idna<3,>=2.5 (from requests<3.0.0,>=2.8.1->conan)
  Downloading https://files.pythonhosted.org/packages/a2/38/928ddce2273eaa564f6f50de919327bf3a00f091b5baba8dfa9460f3a8a8/idna-2.10-py2.py3-none-any.whl (58kB)
Collecting certifi>=2017.4.17 (from requests<3.0.0,>=2.8.1->conan)
  Downloading https://files.pythonhosted.org/packages/c1/6f/3d85f0850962279a7e4c622695d7b3171e95ac65308a57d3b29738b27149/certifi-2020.11.8-py2.py3-none-any.whl (155kB)
Collecting chardet<4,>=3.0.2 (from requests<3.0.0,>=2.8.1->conan)
  Downloading https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl (133kB)
Collecting monotonic>=0.1 (from fasteners>=0.14.1->conan)
  Downloading https://files.pythonhosted.org/packages/ac/aa/063eca6a416f397bd99552c534c6d11d57f58f2e94c14780f3bbf818c4cf/monotonic-1.5-py2.py3-none-any.whl
Collecting packaging (from deprecation<2.1,>=2.0->conan)
  Downloading https://files.pythonhosted.org/packages/46/19/c5ab91b1b05cfe63cccd5cfc971db9214c6dd6ced54e33c30d5af1d2bc43/packaging-20.4-py2.py3-none-any.whl
Collecting MarkupSafe>=0.23 (from Jinja2<3,>=2.9->conan)
  Downloading https://files.pythonhosted.org/packages/4f/8b/da8a2ae5780d38271ac6e691756fc938cf4df8f225eb8aaced0204304932/MarkupSafe-1.1.1-cp38-cp38-win_amd64.whl
Collecting pyparsing>=2.0.2 (from packaging->deprecation<2.1,>=2.0->conan)
  Downloading https://files.pythonhosted.org/packages/8a/bb/488841f56197b13700afd5658fc279a2025a39e22449b7cf29864669b15d/pyparsing-2.4.7-py2.py3-none-any.whl (67kB)
Installing collected packages: PyJWT, idna, certifi, chardet, requests, colorama, PyYAML, patch-ng, monotonic, six, fasteners, node-semver, distro, future, pygments, pyparsing, packaging, deprecation, tqdm, MarkupSafe, Jinja2, python-dateutil, bottle, pluginbase, conan
ERROR: requests 2.24.0 has requirement urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1, but you'll have urllib3 1.26.0 which is incompatible.
  Running setup.py install for patch-ng: started
    Running setup.py install for patch-ng: finished with status 'done'
  Running setup.py install for future: started
    Running setup.py install for future: finished with status 'done'
  Running setup.py install for pluginbase: started
    Running setup.py install for pluginbase: finished with status 'done'
  Running setup.py install for conan: started
    Running setup.py install for conan: finished with status 'done'
Successfully installed Jinja2-2.11.2 MarkupSafe-1.1.1 PyJWT-1.7.1 PyYAML-5.3.1 bottle-0.12.18 certifi-2020.11.8 chardet-3.0.4 colorama-0.4.4 conan-1.31.1 deprecation-2.0.7 distro-1.5.0 fasteners-0.15 future-0.18.2 idna-2.10 monotonic-1.5 node-semver-0.6.1 packaging-20.4 patch-ng-1.17.4 pluginbase-1.0.0 pygments-2.7.2 pyparsing-2.4.7 python-dateutil-2.8.1 requests-2.24.0 six-1.15.0 tqdm-4.51.0
[32;1m$ conan --version[0;m
WARNING: You are using pip version 19.2.3, however version 20.2.4 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.
Traceback (most recent call last):
  File "c:\python38\lib\site-packages\pkg_resources\__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "c:\python38\lib\site-packages\pkg_resources\__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "c:\python38\lib\site-packages\pkg_resources\__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (urllib3 1.26.0 (c:\python38\lib\site-packages), Requirement.parse('urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1'), {'requests'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Python38\Scripts\conan-script.py", line 6, in <module>
    from pkg_resources import load_entry_point
  File "c:\python38\lib\site-packages\pkg_resources\__init__.py", line 3251, in <module>
    def _initialize_master_working_set():
  File "c:\python38\lib\site-packages\pkg_resources\__init__.py", line 3234, in _call_aside
    f(*args, **kwargs)
  File "c:\python38\lib\site-packages\pkg_resources\__init__.py", line 3263, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "c:\python38\lib\site-packages\pkg_resources\__init__.py", line 585, in _build_master
    return cls._build_from_requirements(__requires__)
  File "c:\python38\lib\site-packages\pkg_resources\__init__.py", line 598, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "c:\python38\lib\site-packages\pkg_resources\__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (urllib3 1.26.0 (c:\python38\lib\site-packages), Requirement.parse('urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1'), {'requests'})
section_end:1605099494:step_script
[0Ksection_start:1605099494:cleanup_file_variables
[0K[0K[36;1mCleaning up file based variables[0;m
[0;msection_end:1605099500:cleanup_file_variables
[0K[31;1mERROR: Job failed: exit status 1
[0;m
```

**Comment by czoido** [comment]
This will be closed by: https://github.com/conan-io/conan/pull/8042 already merged in Conan 1.31.2 and will be released in a couple of hours.
For the rest of versions it should not affect after requests launches v2.25.0 (https://github.com/psf/requests/pull/5651) that will be imminent. Meanwhile some of the workarounds mentioned here can be used.
Thanks a lot for reporting!

**Comment by czoido** [comment]
Conan 1.31.2 just released fixing this problem
